server/routes/languages.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const Deck = require('../database/models/Deck');
+const Language = require('../database/models/Language');
+
+const UserLanguage = require('../database/models/UserLanguage');
+
+router.route('/all').get((req, res) => {
+  new Language()
+    .fetchAll({})
+    .then((result) => {
+      return res.send(result.toJSON());
+    })
+    .catch((err) => {
+      console.log('error', err);
+    });
+});
+
+// change user target language
+
+router.route('/target').put((req, res) => {
+  UserLanguage.query({
+    where: { user_id: req.user.id, language_type: 'target' },
+  })
+    .destroy()
+    .then(() => {
```

Comment:
users should be able to have multiple target languages... is this to modify the primary target language?  or to determine what to show on the flashcard translation?